src/pages/Offer/index.test.tsx

```diff
@@ -74,7 +74,6 @@ it('queries when it has a session', async () => {
   await mockNetworkWait()
   wrapper.update()
   expect(wrapper.find(Redirect)).toHaveLength(0)
-  expect(wrapper.find(Price).text()).toBe('99 kr/mån')
```

Comment:
this may be the most important line in this test imo. We should still test this

Reply:
done 👉